data/50-default.json

```diff
@@ -85,5 +85,29 @@
     "ref-kind": "runtime",
     "name": "com.endlessm.CompanionAppService.Locale",
     "branch": "stable"
+  },
+  {
+    "action": "install",
+    "serial": 2019102100,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps",
+    "remote": "eos-apps",
+    "name": "com.hack_computer.Clubhouse",
+    "branch": "stable",
+    "filters": {
+        "~architecture": ["armhf"]
+    }
+  },
+  {
+    "action": "uninstall",
+    "serial": 2019102200,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps.Hack",
+    "remote": "hack-apps",
+    "name": "com.endlessm.Clubhouse",
+    "branch": "stable",
+    "filters": {
+        "~architecture": ["armhf"]
```

Comment:
I don't think the arch filter is needed here. We want the old Clubhouse to be removed on any system it is installed on. It just so happens that we only ever shipped it on x86_64.

Reply:
Yes, removed.